subprojects/docs/src/docs/dsl/org.gradle.api.tasks.bundling.AbstractArchiveTask.xml

```diff
@@ -38,7 +38,7 @@
             </tr>
             <tr>
                 <td>destinationDir</td>
-                <td><literal>project.distsDir</literal></td>
+                <td><literal>project.libsDir</literal></td>
```

Comment:
This isn't correct either.  We have different defaults for Jar/Ear/War and Zip/Tar tasks, but they're all AbstractArchiveTasks.  

Maybe something like this:
```
<td>
<literal>project.distsDir</literal> for <apilink class="org.gradle.api.tasks.bundling.Zip"/> and <apilink class="org.gradle.api.tasks.bundling.Tar"/> tasks and
<literal>project.libsDir</literal> for <apilink class="org.gradle.plugins.ear.Ear"/>, <apilink class="org.gradle.api.tasks.bundling.War"/> and <apilink class="org.gradle.api.tasks.bundling.Jar"/> tasks.
</td>
```